backend/public-graph/graph/resolver.go

```diff
@@ -3434,3 +3444,51 @@ func (r *Resolver) isExcludedError(ctx context.Context, errorFilters []string, e
 	}
 	return false
 }
+
+func (r *Resolver) Login(w http.ResponseWriter, req *http.Request) {
+	var credentials LoginCredentials
+
+	if ADMIN_PASSWORD == "" {
+		http.Error(w, "", http.StatusInternalServerError)
```

Comment:
good idea to check this. could we add a server-side `log.` to report that the password / simple password auth mode is not configured?

Reply:
Done